src/WebApiDebugger/openapi-v1.yaml

```diff
@@ -10,7 +10,7 @@ paths:
       operationId: GetWeatherForecast
       responses:
         '200':
-          description: Success --- New Description
+          description: Success
```

Comment:
Just a thought on this .yaml; is there any gain in having a bit more functionality on the OpenAPI specs on this project so that we can help validate more msbuild generation tasks or is that out of scope for this repo?

Reply:
We should have a spec that is not passing the validation and check that it failed. I will add it to the story to support TreatWarningAsError